src/main/java/com/tw/go/plugin/provider/github/GitHubProvider.java

```diff
@@ -82,4 +77,42 @@ public User getUser(Profile profile) {
         }
         return users;
     }
+
+    @Override
+    public boolean authorize(PluginSettings pluginSettings, User user) {
+        if(StringUtils.isEmpty(pluginSettings.getUsernameRegex())) {
+            return true;
+        } else {
+            return isAMemberOfOrganization(pluginSettings, user);
+        }
+    }
+
+    private boolean isAMemberOfOrganization(PluginSettings pluginSettings, User user) {
+        boolean result = false;
+        try {
+            GitHub github = getGitHub(pluginSettings);
+
+            GHOrganization organization = github.getOrganization(pluginSettings.getUsernameRegex());
```

Comment:
this restricts `UsernameRegex` to have only one organization?
can we support multiple organizations?